biscuit-parser/src/builder.rs

```diff
@@ -387,7 +392,17 @@ impl ToTokens for Binary {
     }
 }
 
-pub type PublicKey = Vec<u8>;
+#[derive(Debug, Clone, Hash, PartialEq, Eq)]
+pub struct PublicKey {
```

Comment:
if we want stricter parsing, we could have an enum with proper `PublicKey` values instead of a `Vec<u8>` and an algorithm enum, but that could be done in another PR